samples/java-valueentity-customer-registry/src/main/java/customer/Main.java

```diff
@@ -22,14 +22,22 @@
   // tag::register[]
   public static AkkaServerless createAkkaServerless() {
     return AkkaServerlessFactory.withComponents(
-      CustomerValueEntity::new,
       // end::register[]
-      CustomerSummaryByNameView::new,
-      CustomersResponseByNameView::new,
-      CustomerByEmailView::new,
+      CustomerValueEntity::new,
       CustomerActionImpl::new,
+      CustomerByEmailView::new,
+      CustomerByNameView::new,
+      CustomerSummaryByNameView::new,
+      CustomersResponseByNameView::new);
+      /*
+      // tag::register[]
+      return AkkaServerlessFactory.withComponents(
+        CustomerValueEntity::new,
+        CustomerByNameView::new);
+      // end::register[]
```

Comment:
Maybe this one should have its own order, so the code snippet is still part of what gets compiled?